### PR TITLE
fix the problems when install viper in Ubuntu16.04

### PR DESCRIPTION
**Pull request by kaikai1024**
### - What I did
little doc fix
### - How I did it
find it when i install viper
### - How to verify it
i don't know if it is ok while os is not Ubuntu16.04
### - Description for the changelog
fix it when install python3.6 and use virtualenv
### - Cute Animal Picture
: -D
> put a cute animal picture here.

### CODE REVIEW

File: docs/installing-viper.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ other development environment set-up.
 
 To create a new virtual environment for Viper run the following commands:
 ::
-    virtualenv -p /usr/local/lib/python3.6/bin/python --no-site-packages ~/viper-venv
+    virtualenv -p /usr/local/lib/python3.6/bin/python3 --no-site-packages ~/viper-venv
```
Comment (fubuloubu) [inline docs/installing-viper.rst]: I think this is my bad, I created a symlink `python` pointing to `python3` I think got copied when I showed the original author my install instructions for 16.04.LGTM

Comment (kaikai1024) [reply docs/installing-viper.rst]: And i wonder if it's better to add some examples just like the doc of the solidity
[https://solidity.readthedocs.io/en/develop/solidity-by-example.html](example of the solidity)

Comment (fubuloubu) [reply docs/installing-viper.rst]: We have examples in a separate section we are working on integrating into the docs

Comment (kaikai1024) [reply docs/installing-viper.rst]: cool

**Comment by DavidKnott** [comment]
LGTM

**Pull request merged**